docs/user/alerting/rule-types.asciidoc

```diff
@@ -74,6 +74,11 @@ beta:[] {ml-docs}/ml-configuring-alerts.html[{ml-cap} rules] run scheduled check
 
 Security rules detect suspicious source events with pre-built or custom rules and create alerts when a rule’s conditions are met. For more information, refer to {security-guide}/prebuilt-rules.html[Security rules].
 
+[NOTE]
+==============================================
+Alerts associated with Security rules are not visible in the stack management view.
```

Comment:
> Alerts associated with Security rules are not visible in the stack management view.

Which stack management app in particular does this limitation apply to? Do you mean this one?:

![image](https://user-images.githubusercontent.com/26471269/163287533-0f16fed7-fe84-4e3d-b412-b4403f595950.png)

Reply:
yes - do we have a {var} I should use to dynamically cite the correct name?  Or a better static desired text?  Any suggestions welcome, thanks Lisa

Reply:
I've added a suggestion.  In my quick tests it seemed like the same limitation might also be true for observability rules (but not if the rule is created in the Stack Management app?).

Reply:
I'm not sure on that nuance.  @mikecote can you help confirm?

Reply:
or maybe @kobelb knows this?  He gave me tips on the Security vs Platform alerts originally.

Reply:
I'm not aware of such limitations, @XavierM may know more.

Reply:
I'm not aware of or seeing this same limitation with the observability rules.

@lcawl was there a specific o11y rule that you were seeing this behavior?

Reply:
Last week I did a quick test with the "rules" scenario in the MLQA framework on 8.3. I could see what looked like alerts in the Observability app:
![image](https://user-images.githubusercontent.com/26471269/163873904-2dfc1660-c93d-47ac-b42d-0ed9a8015786.png)

... but no such info in the Stack Management app:
![image](https://user-images.githubusercontent.com/26471269/163873699-253b02a9-723c-4151-be92-2f5a20ba3bca.png)
...so I assumed it was the same limitation.

Reply:
Xavier mentioned privately that none of them show in Stack Management - so this isn't specific to Security only.  should we update to say 'Security or Observability' then?  @lcawl thoughts?

Reply:
I've added a note to the "Observability rules" section and I've also drafted a new "Limitations" section in the troubleshooting docs.